docs/how_to/data_intake/cache.md

```diff
@@ -0,0 +1,69 @@
+# How to use cache
+
+:::{admonition} What does this guide solve?
+:class: important
+This guide will go through how to cache large files for easier access.
+:::
+
+## Caching file
+When working with large files, it can be very advantageous to cache it and therefore be able to save time when reloading the data.
+To cache a [source](../../reference/source) is done by using `cache_dir` followed by a directory.
+If the directory does not exist, it will be created.
+Below is an example of caching a 370 MB file, and the table consists of almost 12 million rows.
+
+:::{warning}
+The initial load can take a couple of minutes as the file needs to be downloaded and cached first.
+:::
+
+::::{tab-set}
+:::{tab-item} Specification (YAML)
```

Comment:
I don't really like the names `Specification (YAML)` and `Pipeline (Python)`.. there is a concept of specification in Python, and a pipeline in the YAML interface.. so it gets confusing..

I would just call it `YAML` and `Python`

Reply:
```suggestion
:::{tab-item} YAML
```